docs/data/joy/main-features/color-inversion/ColorInversionNavigation.js

```diff
@@ -30,24 +31,23 @@ export default function ColorInversionNavigation() {
     <Box sx={{ display: 'flex', borderRadius: 'sm', overflow: 'auto' }}>
       <Sheet
         variant="solid"
-        color="neutral"
         invertedColors
         sx={{
           p: 2,
           ...(color !== 'neutral' && {
-            bgcolor: `${color}.800`,
+            bgcolor: `${color}.700`,
           }),
         }}
       >
         <Select
-          variant="outlined"
-          defaultValue="1"
+          variant="soft"
           size="sm"
+          defaultValue="1"
           placeholder={
-            <div>
-              <Typography level="inherit">Saleshouse</Typography>
+            <Stack alignItems="start">
```

Comment:
I guess we can remove the placeholder altogether if there's always a selected option by default — clean it up a bit.